templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/list_statements.html.twig

```diff
@@ -1,10 +1,7 @@
 {% extends '@DemosPlanCore/DemosPlanCore/base.html.twig' %}
 {% block component_part %}
 
-    {% set path = hasPermission('feature_statements_import_excel') or
-        hasPermission('feature_import_statement_pdf') or
-        hasPermission('feature_segments_import_excel') or
-        hasPermission('feature_simplified_new_statement_create') ? 'DemosPlan_procedure_import' : 'DemosPlan_statement_new_submitted' %}
+    {% set path = hasPermission('feature_simplified_new_statement_create') ? 'DemosPlan_procedure_import' : 'DemosPlan_statement_new_submitted' %}
```

Comment:
I'm not sure, but shouldn't it be
`
(hasPermission('feature_simplified_new_statement_create') && (hasPermission('feature_import_statement_pdf') || hasPermission('feature_segments_import_excel'))
`
?
Or is there no case in which there's only the new statement create form and no import?